docs/API.md

```diff
@@ -1,3 +1,36 @@
+# API documentation
+
+## Getting an API token
+
+To get access to the API, you should register your app and obtain a
+token. To do so, use the `apps` API (accessible without a token) to create
```

Comment:
Just "use apps" is maybe a bit unclear, maybe you should use the full path, and also tell how to use it (Do a post with such data, and such other is optional and....)
```suggestion
token. To do so, use the `/api/v1/apps` API (accessible without a token) to create
```

Reply:
The request should be documented with Swagger (but it is broken for the moment :cry:)